join_request/joepetrowski.yml

```
@@ -0,0 +1,24 @@
+collective: Potoc
+member:
+  name: Joseph Petrowski
+  address: 15DCWHQknBjc5YPFoVj8Pn2KoqrqYywJJ95BYNYJ4Fj3NLqz
+  github: joepetrowski
```

please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  address: 15DCWHQknBjc5YPFoVj8Pn2KoqrqYywJJ95BYNYJ4Fj3NLqz
  matrix:
  github: joepetrowski
```

---

```suggestion
  address: 15DCWHQknBjc5YPFoVj8Pn2KoqrqYywJJ95BYNYJ4Fj3NLqz
  github: joepetrowski
```
```suggestion
  address: 15DCWHQknBjc5YPFoVj8Pn2KoqrqYywJJ95BYNYJ4Fj3NLqz
  matrix: @joe:web3.foundation
  github: joepetrowski
```